tests/integration_tests/config/derive_mirror_rnda_psf_measurement.yml

```diff
@@ -2,26 +2,19 @@
 applications:
 - application: simtools-derive-mirror-rnda
   configuration:
-    containment_fraction: 0.8
     log_level: DEBUG
     model_version: 6.0.2
-    output_file: test_derive_mirror_rnda_psf_measurement.ecsv
     output_path: simtools-output
-    psf_measurement: ./tests/resources/MLTdata-preproduction.ecsv
-    rnda: 0.0063
-    rtol_psf_containment: 0.1
+    data: tests/resources/PSFcurve_data_v2.ecsv
+    mirror_list: tests/resources/mirror_list_CTA-N-LST1_v2019-03-31_rotated.ecsv
     site: North
-    telescope: MSTx-NectarCam
+    telescope: LSTN-01
     test: true
-    use_random_focal_length: true
+    use_random_focal_length: false
   integration_tests:
```

Comment:
Wondering if we want to test that the output values (three model parameter values) is consistent with expectations? This would require to put an example json file into tests/resources/model_parameters and add a reference_output_file point like in tests/integration_tests/config/db_get_parameter_from_db_telescope_parameter_version.yml (don't ask about the formatting in this file)